Model/Api/CreateOrder.php

```diff
@@ -171,7 +180,7 @@ public function execute(
         $currency = null
     ) {
         try {
-
+            $startTime = round(microtime(true) * 1000);
```

Comment:
up to you but we could have simple timer class here for better redability. Timer would have two method, startTimer() and getElapsedTime()

Reply:
Adding a timer and a metric class. I am putting them in the helper function classes. Do we have a separate directory I should be adding these in?

Reply:
Helper directory seems good